core/src/main/java/io/questdb/cairo/wal/WalWriter.java

```diff
@@ -105,6 +105,7 @@ public class WalWriter implements TableWriterAPI {
     private long txnMinTimestamp = Long.MAX_VALUE;
     private boolean txnOutOfOrder = false;
     private int walLockFd = -1;
+    private long seqTxn = NO_TXN;
```

Comment:
I'd rename it 
```suggestion
    private long lastSeqTxn = NO_TXN;
```

Reply:
fixed in https://github.com/questdb/questdb/pull/3637/commits/c70d5416fb61308635c748fb6ce8ef554098bc95